crates/holochain_integrity_types/src/dna_modifiers.rs

```diff
@@ -28,12 +26,6 @@ pub struct DnaModifiers {
     /// All Action timestamps must come after this time.
     #[cfg_attr(feature = "full-dna-def", builder(default = "Timestamp::now()"))]
     pub origin_time: Timestamp,
-
-    /// The smallest unit of time used for gossip time windows.
-    /// You probably don't need to change this.
-    #[cfg_attr(feature = "full-dna-def", builder(default = "standard_quantum_time()"))]
-    #[cfg_attr(feature = "full-dna-def", serde(default = "standard_quantum_time"))]
-    pub quantum_time: Duration,
```

Comment:
Removing quantum_time from dna modifiers and all follow on types such as dna bundling types.